handlers/handler_test.go

```diff
@@ -163,3 +163,46 @@ func TestGetConfigInfoUsingExtraServiceInfo(t *testing.T) {
+		{"dg.4825/da85ab42-53a0-4698-9b38-7ad59b770b47/flies/moth.txt", "dg.4825/da85ab42-53a0-4698-9b38-7ad59b770b47", "flies/moth.txt"},
+		{"dg.F738/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test1.txt", "dg.F738/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test1.txt"},
+		{"dg.80B6/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test2.txt", "dg.80B6/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test2.txt"},
+		{"dg.EA80/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test3.txt", "dg.EA80/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test3.txt"},
+		{"dg.5B0D/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test4.txt", "dg.5B0D/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test4.txt"},
+		{"dg.4503/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test5.txt", "dg.4503/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test5.txt"},
+		{"dg.373F/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test6.txt", "dg.373F/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test6.txt"},
+		{"dg.7519/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test7.txt", "dg.7519/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test7.txt"},
+		{"dg.7C5B/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test8.txt", "dg.7C5B/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test8.txt"},
+		{"dg.0896/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test9.txt", "dg.0896/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test9.txt"},
+		{"dg.F82A1A/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test10.txt", "dg.F82A1A/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test10.txt"},
+		{"dg.4DFC/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test11.txt", "dg.4DFC/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test11.txt"},
+		{"dg.712C/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test12.txt", "dg.712C/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test12.txt"},
+		{"dg.6VTS/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test13.txt", "dg.6VTS/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test13.txt"},
+		{"dg.63D5/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test14.txt", "dg.63D5/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test14.txt"},
+		{"dg.414F/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test15.txt", "dg.414F/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test15.txt"},
+		{"dg.6539/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test16.txt", "dg.6539/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test16.txt"},
+		{"dg.ANV0/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test17.txt", "dg.ANV0/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test17.txt"},
+		{"dg.MD1R/da85ab42-53a0-4698-9b38-7ad59b770b47/files/prefixtest/test18.txt", "dg.MD1R/da85ab42-53a0-4698-9b38-7ad59b770b47", "files/prefixtest/test18.txt"},
```

Comment:
I think we need tests for guids without prefixes and also filenames without subdirectories.

Reply:
Added more scenarios with this.

Reply:
Can we add a test for a key not having a prefix but having a subdirectory in the filename? I'm not sure that's accounted for.